viam-cartographer/src/slam_service/slam_service.cc

```diff
@@ -74,24 +74,26 @@ ::grpc::Status SLAMServiceImpl::GetPosition(ServerContext *context,
 
     // Rotate pose to XZ plane. Additional angle offset is used so rotations
     // occur along the Y axis, to match the XZ plane
-    auto rotated_vector = pcdRotation * global_pose.translation();
-    auto rotated_quat =
-        pcdOffsetRotation * global_pose.rotation() * pcdRotation;
+    // auto rotated_vector = pcdRotation * global_pose.translation();
+    // auto rotated_quat = pcdOffsetRotation * global_pose.rotation() *
+    // pcdRotation;
+    auto pos_vector = global_pose.translation();
+    auto pos_quat = global_pose.rotation();
 
     // Set pose for our response
     Pose *myPose = response->mutable_pose();
-    myPose->set_x(rotated_vector.x());
-    myPose->set_y(rotated_vector.y());
-    myPose->set_z(rotated_vector.z());
+    myPose->set_x(pos_vector.x());
```

Comment:
Are we sure that nothing needs to change on the `GetPosition` endpoint?

Reply:
Rotation of the position was removed on lines 77-78